db/migrate/20180225173740_migrate_deprecated_kinds_teams.rb

```diff
@@ -0,0 +1,33 @@
+class MigrateDeprecatedKindsTeams < ActiveRecord::Migration[5.1]
+  class Team < ApplicationRecord
+    self.table_name = 'teams'
```

Comment:
Minor note here: I've understood that it's better to use `connection.execute` with an SQL statement instead of using the `ActiveRecord` functionality. Is this overkill?

Reply:
Hm, since I redefine the class with its table name, using AR is generally "save" in this migration.

But you're of course right, I'd also prefer using the raw connection. I just thought it may be harder to read and less in the line of the existing migrations... But I'll see how it looks like 👍